ParisBNF/abb/BNFabb147.xml

```diff
@@ -159,11 +160,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                             ብዙ<cb n="b"/><hi rend="rubric">ም፡ ሴቶች፡ መጹ፡ ነው፡</hi> ይቤ፡ ማለት፡ አለ፨
                             ሖ<hi rend="rubric">ረ፡ ማ፡ ሔደ። አስተር</hi>አየ፡ ተርእየ፡ ማ፡ ታየ፨
                          </incipit>
-                         <note>Starting from <locus target="18">page 18</locus>, the text is divided into
-                         chapters (<foreign xml:lang="gez">አንቀጽ፡</foreign>) is introduced
-                         (but the numbering starts from chapter 22 up to chapter 68).
+                         <note>Starting from <locus target="#9v">18</locus>, the text is divided into
+                         chapters (called <foreign xml:lang="gez">አንቀጽ፡</foreign>)
+                         but the numbering starts from chapter 22 (up to chapter 68).
                        The name of the owner of commissioner at the end of the text (written over an erasure),
-                       on <locus target="122b">page 122b</locus>, has been erased and is nearly unreadable,
+                       on <locus target="#64vb">122b</locus>, has been erased and is nearly unreadable,
```

Comment:
122 stands for f. 61 if we start from 1 and 63 if we start from 3 (if i-ii become 1-2), surely never 64